Phonebook.Frontend/src/app/modules/table/dialogs/table-settings-dialog/table-settings.dialog.html

```diff
@@ -14,6 +14,7 @@ <h1 mat-dialog-title>
   <p i18n="TableSettingsDialog|Description of how to configure the table Settings@@TableSettingsDialogDescription">
     Drag and Drop the Columns in order to arrange them in an order you like. You can hide them as well by dragging them
     to the right.
+    <b>Note: You can only search for visible columns.</b>
```

Comment:
```suggestion
    <b>Note: You can only search for displayed columns.</b>
```

I would rename this because the word "displayed" will also be used down under.

Reply:
I am against it :)